RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -108,21 +109,16 @@ class VoiceBroadcastPlaybackViewModel: VoiceBroadcastPlaybackViewModelType, Voic
         displayLink.isPaused = false
         isActuallyPaused = false
         
-        if voiceBroadcastAggregator.isStarted == false {
-            // Start the streaming by fetching broadcast chunks
-            // The audio player will automatically start the playback on incoming chunks
-            MXLog.debug("[VoiceBroadcastPlaybackViewModel] play: Start streaming")
-            state.playbackState = .buffering
-            voiceBroadcastAggregator.start()
-        } else if let audioPlayer = audioPlayer {
+        if let audioPlayer = audioPlayer {
             MXLog.debug("[VoiceBroadcastPlaybackViewModel] play: resume")
             audioPlayer.play()
-        } else {
+        } else if voiceBroadcastAggregator.launchState == .loaded {
```

Comment:
This check "voiceBroadcastAggregator.launchState == .loaded" should be done after the playbackState change

We have to update the playbackState with .buffering to store the play request
Then if the aggregator is loaded, we restart from the beginning

Reply:
Fixed in last commit.